lib/enkf/local_config.cpp

```diff
@@ -138,11 +138,7 @@ static local_updatestep_type * local_config_alloc_updatestep( local_config_type
 
 
 void local_config_clear( local_config_type * local_config ) {
-  local_config->default_updatestep  = NULL;
   hash_clear( local_config->updatestep_storage );
-  hash_clear( local_config->ministep_storage );
```

Comment:
I dont understand why `clear` is not clearing anymore

Reply:
Well - the `update_step` is the interesting for waht is the currently active config - that is cleared. The others are retained so that they can - optionally - be reused as named objects in a different to-be-defined configuration.